pkg/resources/syslogng/service.go

```diff
@@ -99,10 +99,12 @@ func (r *Reconciler) monitorServiceMetrics() (runtime.Object, reconciler.Desired
 				TargetLabels:    nil,
 				PodTargetLabels: nil,
 				Endpoints: []v1.Endpoint{{
-					Port:          "http-metrics",
-					Path:          "/metrics",
-					Interval:      "15s",
-					ScrapeTimeout: "5s",
+					Port:           "http-metrics",
```

Comment:
Should I add the `TLSConfig` and `Scheme` fields here as well? They are not present in `func monitorBufferServiceMetrics()` either.

Reply:
yes please, all the options should be available everywhere as long as we are talking about the same type and I beleive we do